rules/phpunit/src/Rector/MethodCall/DelegateExceptionArgumentsRector.php

```diff
@@ -33,9 +33,9 @@ public function getDefinition(): RectorDefinition
                 new CodeSample(
                     '$this->setExpectedException(Exception::class, "Message", "CODE");',
                     <<<'PHP'
-$this->setExpectedException(Exception::class);
-$this->expectExceptionMessage("Message");
-$this->expectExceptionCode("CODE");
+$this->setExpectedException(Throwable::class);
```

Comment:
I am not sure why the ecs change `Exception::class` to `Throwable::class` on this, @TomasVotruba could you verify?

Reply:
There is a rule that does this change as part of upgrade to PHP 7.0.

Could you update the line above also to heredoc?
That way the change would be in both snippets and correct

Reply:
I've updated to heredoc

Reply:
Do you think `Throwable::class` need to be reverted to `Exception::class` ?

Reply:
I reverted to nowdoc and changed back to `Exception::class`

Reply:
Now all green, I reverted to nowdoc as phpstan complain about it.